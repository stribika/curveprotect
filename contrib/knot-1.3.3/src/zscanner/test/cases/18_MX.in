$ORIGIN	.
$TTL	1

; For more tests on dname see NS test (same processing)

; OK
@	MX	0	@		; Minimal priority
@	MX	65535	@		; Maximal priority
@	MX	1	mail		; Relative dname
@	MX	1	mail.tld.	; Absolute dname
@	MX	\# 3 0001 00		; Hexadecimal rdata
@	TYPE15	\# 3 0001 00		; TYPE + Hexadecimal rdata
@	TYPE15	1	@		; TYPE
@	mx	1	@		; Type in lower-case

; KO
@	MX
@	MX				; Empty rdata
@	MX	\# 0			; Hex empty rdata
@	MX	\#			; Missing hex length
@	MX	-1	@		; Negative number
@	MX	65536	@		; 16bit overflow
@	MX	1	$		; Bad dname
@	MX	0	@ x		; Unexpected item
@	MX	\# 4 0001 0001		; Too long rdata
@	MX	\# 4 0001 00		; Bad rdata length
