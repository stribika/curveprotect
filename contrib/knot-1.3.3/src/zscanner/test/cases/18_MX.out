OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0000 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=FFFF 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 046D61696C00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 046D61696C03746C6400
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=000F
RDATA=0001 00
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_RDATA
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
